gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts

```
@@ -280,6 +284,9 @@ tasks {
 	checkstyleTest {
 		config = resources.text.fromFile(checkstyle.configDirectory.file("checkstyleTest.xml"))
 	}
+	check {
+		dependsOn("rewriteDryRun")
```

```suggestion
		dependsOn("failOnDryRunResults")
```

- https://github.com/openrewrite/rewrite-gradle-plugin/issues/385#issuecomment-3046297673